python/dgl/dataloading/dataloader.py

```diff
@@ -64,7 +64,8 @@ def _next_indices(self):
     def __next__(self):
         batch = self._next_indices()
         if self.mapping_keys is None:
-            return batch
+            # clone() fixes #3755.  Not sure why.  Need to take a look afterwards.
+            return batch.clone()
```

Comment:
Is batch a Torch Tensor created by PyTorch?

Reply:
It's a view of `self.dataset`, which in turn comes from https://github.com/dmlc/dgl/blob/a05a624d8f4820a8755d75a0e7a81eb04ec19de8/python/dgl/dataloading/dataloader.py#L141-L144.